tests/api/basic.hurl

```diff
@@ -1,9 +1,13 @@
-GET http://{{host}}:4000/api/tickets/0
+GET http://{{host}}:4000/api/tickets/1
 Content-Type: application/json
 [Cookies]
 user: acmecorp / alice
 HTTP 200
-{"id":"0","customer":"Globex","last_updated":"2022-07-19T15:39:53Z","description":"Dooms day device needs to be refactored","resolved":false}
```

Comment:
`last_updated` no longer is hardcoded, so I've loosened the assertions up a bit.